tests/test_masks.py

```diff
@@ -2,7 +2,6 @@
 
 import numpy as np
 import pytest
-
```

Comment:
okay.... `pre-commit run --all` on my machine adds this line, then the bot removes it.

Reply:
well, the bot didn't complain about it in a578c40. maybe it's related to the import failure somehow?